storage/storage-mq/src/util/postgresRepository.ts

```diff
@@ -0,0 +1,76 @@
+import { Pool, PoolConfig, PoolClient, QueryResult } from 'pg'
+
+export default class PostgresRepository {
+  private connectionPool?: Pool = undefined
+
+  /**
+   * Initializes the connection to the database.
+   * @param retries:  Number of retries to connect to the database
+   * @param backoffMs:  Time in ms to backoff before next connection retry
+   * @returns reject promise on failure to connect
+   */
+  public async init (poolConfig: PoolConfig, retries: number, backoffMs: number): Promise<void> {
+    console.debug('Initializing databse connection')
```

Comment:
The first `console.debug()` is not useful. The second `console.debug()` contains all the scarceness information. 
Therefore, I would remove the first one.